app/HutchManager/Services/RquestPollingService.cs

```diff
@@ -74,13 +80,14 @@ private async void PollRquest(object? state)
                     RunTimerOnce();
                     return;
                 }
-      var aSource = await _db.ActivitySources
-            .AsNoTracking()
-            .Include(x => x.Type)
-            .Where(x => x.Id == job.ActivitySourceId)
-            .SingleOrDefaultAsync()
-          ?? throw new KeyNotFoundException();
-      SendToQueue(job,aSource.TargetDataSourceName);
+                
+                var aSource = await _db.ActivitySources
+                                  .AsNoTracking()
+                                  .Include(x => x.Type)
+                                  .Where(x => x.Id == job.ActivitySourceId)
+                                  .SingleOrDefaultAsync()
+                                ?? throw new KeyNotFoundException();
+                SendToQueue(job,aSource.TargetDataSourceName);
```

Comment:
this probably wants to be awaited now it's `async`?
```suggestion
                await SendToQueue(job,aSource.TargetDataSourceName);
```